test/processes_mysql.jl

```diff
@@ -51,46 +52,89 @@ end
 @testset "MESH2UMLS" begin
     # $TRAVIS is equal to "true" if it's a Travis build, and "false" if it's not.
     is_travis = lowercase(strip(get(ENV, "TRAVIS", ""))) == "true"
+
     # $TRAVIS_PULL_REQUEST is equal to the PR number if it is a PR, and "false" if it's not.
     is_travis_pull_request = lowercase(strip(get(ENV, "TRAVIS_PULL_REQUEST", ""))) != "false"
+
     # If this is a Travis build, only execute this test if it is NOT a pull request.
     # If this is not a Travis build, then always execute this test.
     if !is_travis || !is_travis_pull_request
```

Comment:
Could we simplify this to only check travis_pull_request?  Since if it's travis or local we always wan't to execute.

Reply:
Done!